polygon/bridge/db.go

```diff
@@ -17,10 +17,51 @@ var databaseTablesCfg = kv.TableCfg{
 	kv.BorEventNums: {},
 }
 
+type Store interface {
+	Prepare(ctx context.Context) error
+	Close()
+
+	GetLatestEventID(ctx context.Context) (uint64, error)
+	GetSprintLastEventID(ctx context.Context, lastID uint64, timeLimit time.Time, stateContract abi.ABI) (uint64, error)
+	AddEvents(ctx context.Context, events []*heimdall.EventRecordWithTime, stateContract abi.ABI) error
+	GetEvents(ctx context.Context, start, end uint64) ([][]byte, error)
+	StoreEventID(ctx context.Context, eventMap map[uint64]uint64) error
+	GetEventIDRange(ctx context.Context, blockNum uint64) (uint64, uint64, error)
+	PruneEventIDs(ctx context.Context, blockNum uint64) error
+}
+
+type store struct {
+	db *polygoncommon.Database
+}
+
+func NewStore(db *polygoncommon.Database) Store {
```

Comment:
Let's stick to Go's idiology of always returning concrete structs instead of interfaces and use interfaces only for dependencies (in constructors) or function inputs.

With this in mind, I would suggest renaming the `store` struct to `MdbxStore` and `NewMdbxStore(...) *MdbxStore` - this is matching the style in `polygon/heimdall Service` which is nice